src/app/components/client/toolbar/UserMenu.tsx

```diff
@@ -4,24 +4,214 @@
+import SignOutIcon from "./images/menu-icon-signout.svg";
+
+export type UserMenuProps = {
   user: Session["user"] | null;
 };
 
-export const UserMenu = (props: Props) => {
-  // Placeholder to enable login/logout until we create an actual menu
-  // Ignored for test coverage; to be implemented:
-  /* c8 ignore next 3 */
+export const UserMenu = (props: UserMenuProps) => {
+  const l10n = useL10n();
+
+  const fxaItemRef = useRef<HTMLAnchorElement>(null);
+  const settingsItemRef = useRef<HTMLAnchorElement>(null);
+  const helpItemRef = useRef<HTMLAnchorElement>(null);
+  const signOutItemRef = useRef<HTMLButtonElement>(null);
+
   if (!props.user) {
-    return <button onClick={() => void signIn("fxa")}>Sign in</button>;
+    return <SignInButton />;
   }
 
+  const itemKeys = {
+    fxa: "fxa",
+    settings: "settings",
+    help: "help",
+    signout: "signout",
+  };
+
+  const handleOnAction = (menuItemKey: Key) => {
+    switch (menuItemKey) {
+      case itemKeys.fxa:
+        fxaItemRef.current?.click();
+        break;
+      case itemKeys.settings:
+        settingsItemRef.current?.click();
+        break;
+      case itemKeys.help:
+        helpItemRef.current?.click();
+        break;
+      case itemKeys.signout:
+        signOutItemRef.current?.click();
+        break;
+      default:
+      // do nothing
+    }
+  };
+
   return (
-    // Ignored for test coverage; to be implemented:
-    /* c8 ignore next */
-    <button onClick={() => void signOut({ callbackUrl: "/" })}>Sign out</button>
+    <MenuTrigger user={props.user} onAction={handleOnAction}>
+      <Item
+        key={itemKeys.fxa}
+        textValue={l10n.getString("user-menu-manage-fxa-label")}
+      >
+        <b>{props.user.email}</b>
+        <a
+          className={styles.menuItemCta}
+          href={process.env.NEXT_PUBLIC_FXA_SETTINGS_URL}
+          ref={fxaItemRef}
+          target="_blank"
+        >
+          {l10n.getString("user-menu-manage-fxa-label")}
+          <Image src={OpenInIcon} alt="" height={24} width={24} />
+        </a>
+      </Item>
+      <Item
+        key={itemKeys.settings}
+        textValue={l10n.getString("user-menu-settings-label")}
+      >
+        <Link
+          className={styles.menuItemCta}
+          href="/redesign/user/settings/"
+          ref={settingsItemRef}
+          title={l10n.getString("user-menu-settings-tooltip")}
+        >
+          <Image src={SettingsIcon} alt="" height={24} width={24} />
+          {l10n.getString("user-menu-settings-label")}
+        </Link>
+      </Item>
+      <Item
+        key={itemKeys.help}
+        textValue={l10n.getString("user-menu-help-label")}
+      >
+        <a
+          className={styles.menuItemCta}
+          href={process.env.NEXT_PUBLIC_EXTERNAL_SUPPORT_URL}
+          ref={helpItemRef}
+          rel="noopener noreferrer"
+          target="_blank"
+          title={l10n.getString("user-menu-help-label-tooltip")}
```

Comment:
I don't see a matching key in the .ftl files for this.
I... think this is related to my other comment re: this unused string,

```ini
user-menu-help-tooltip = Get help using { -brand-fx-monitor }
```

```suggestion
          title={l10n.getString("user-menu-help-tooltip")}
```

Reply:
Resolved with https://github.com/mozilla/blurts-server/pull/3326#discussion_r1307330011.